src/cmd/provisioner/run.go

```diff
@@ -63,6 +63,7 @@ func (r *Run) validate() error {
 
 // Execute implements subcommands.Command.Execute.
 func (r *Run) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{}) subcommands.ExitStatus {
+	deps := args[0].(provisioner.Deps)
```

Comment:
just curious to know if we need any checks here? I understand that it is only being called from main.go:57 right now with the appropriate args.

Reply:
If the type assertion fails, the program will bail with a panic. I think this is fine, since this is a "this should absolutely never happen" kind of situation. If main.go is modified to use inappropriate args, we should see the panic in tests.